code/modules/mob/living/simple_animal/abnormality/waw/sphinx.dm

```diff
@@ -0,0 +1,472 @@
+//Coded by Coxswain
+/mob/living/simple_animal/hostile/abnormality/sphinx
+	name = "Sphinx"
+	desc = "A gigantic stone feline."
+	icon = 'ModularTegustation/Teguicons/64x64.dmi'
+	icon_state = "sphinx"
+	icon_living = "sphinx"
+	var/icon_aggro = "sphinx_eye"
+	faction = list("hostile")
+	speak_emote = list("intones")
+	pixel_x = -16
+	base_pixel_x = -16
+	ranged = TRUE
+	maxHealth = 2000
+	health = 2000
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 1.2, WHITE_DAMAGE = 0.5, BLACK_DAMAGE = 0.8, PALE_DAMAGE = 1.5)
+	stat_attack = HARD_CRIT
+	speed = 4
+	move_to_delay = 4
+	melee_damage_lower = 70
+	melee_damage_upper = 70
+	attack_sound = 'sound/abnormalities/sphinx/attack.ogg'
+	can_breach = TRUE
+	threat_level = WAW_LEVEL
+	melee_damage_type = WHITE_DAMAGE
+	armortype = WHITE_DAMAGE
+	melee_damage_lower = 100
+	start_qliphoth = 3
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = 35,
+		ABNORMALITY_WORK_INSIGHT = 40,
+		ABNORMALITY_WORK_ATTACHMENT = 0,
+		ABNORMALITY_WORK_REPRESSION = 15,
```

Comment:
Levels are entirely unused? Level 1 agent will have nearly the same chances as level 5, ignoring temperance buff.